desktop/menu.ts

```diff
@@ -0,0 +1,77 @@
+import { app, Menu, MenuItemConstructorOptions } from 'electron'
+
+const isMac = process.platform === 'darwin'
+
+const template = [
+    ...(isMac ? [{
+        label: app.name,
+        submenu: [
+            { role: 'about' },
+            { type: 'separator' },
+            { type: 'separator' },
+            { role: 'quit' }
+        ]
+    }]: []) as MenuItemConstructorOptions[],
+    // { role: fileMenu }
+    {
+        label: 'Fileyooo',
+        submenu: [
+            {
+                label: 'Open file',
+                click: async () => {
+                    //doOpenFile()
```

Comment:
@romicolman this is the way of making a submenu button execute an action from within the app. It could be anything from within the app.